devices/Adxl343/README.md

```diff
@@ -0,0 +1 @@
+# nanoADXL343
```

Comment:
please look at the other readme and please fill it properly.
Also add the file category